new_platforms/src/Trusted/optee/report_optee.c

```diff
@@ -31,6 +32,43 @@ oe_result_t oe_verify_report(
     size_t report_size,
     oe_report_t* parsed_report)
 {
-    /* Not supported */
-    return OE_UNSUPPORTED;
+    oe_result_t oeResult = OE_OK;
+    if (parsed_report != NULL)
+    {
+        oeResult = oe_parse_report(report, report_size, parsed_report);
```

Comment:
I am thinking about calling oe_parse_report (or subset of) here no matter to validate that TCPS blob is present and valid. Let me know if you think we should or should not do that here.

Reply:
I don;t know what that means.  But parse_report is not used below this line, so currently this call is not useful for anything as far as I can tell.

Reply:
oe_parse_report is validating that the TCPS blob OID is present and well formed. So if oe_parse_report fails, we know that verify should fail too. Ultimately this is about what should we check in _verify.

Reply:
I have refactored the code to not have to parse the PEM multiple times.